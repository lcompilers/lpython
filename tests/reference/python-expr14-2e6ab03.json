{
    "basename": "python-expr14-2e6ab03",
    "cmd": "lpython --no-color --show-python {infile}",
    "infile": "tests/expr14.py",
    "infile_hash": "d18abc152939c944f6a1e493754cea8b72f0d03d360ab4c9e75ad697",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "python-expr14-2e6ab03.stdout",
    "stdout_hash": "52e647db794a391cb9ebd5978bb97de935a62e1b85d890232b1a878a",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}