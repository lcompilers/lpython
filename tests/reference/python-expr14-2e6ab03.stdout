def test_boolean_comparison():
    b1: bool
    b2: bool
    b3: bool
    b1 = True
    b2 = True
    b3 = False
    b1 = (b1) > (b2)
    b1 = (b1) == (b2)
    b1 = (b2) != (b3)
    b1 = (b2) >= (b3)
    b1 = (b2) <= (b3)
